### PR TITLE
Fast js updates

### PR DESCRIPTION
**Pull request by fasterSean**
# Updates
* Add new information for FastJS @brikr 
* Remove dev rel stuff https://fastaf.slack.com/archives/C014VRDRUC8/p1627416269247700
* Remove discord https://fastaf.slack.com/archives/C01J1G8NAAW/p1627415900290500

### CODE REVIEW

**Review by zjoekr — APPROVED** [review]

File: index.mdx [hunk 1]
```diff
@@ -58,30 +58,28 @@ export default LandingLayout;
...
-      <ThinTile to="developer-portal/index.md" icon={docs} header="Docs & Guides">
-        Dive into our documentation & guides for integrating Fast with your platform. 
+    <ThinTile to="developer-portal/index.md" icon={docs} header="Docs & Guides">
+      Dive into our documentation & guides for integrating Fast with your
+      platform.
     </ThinTile>
-    <ThinTile to="https://community.fastdevs.co" icon={community} header="Community">
-        Ask for help, share your feedback, or ask about special use-cases in our community.
+    <ThinTile
+      to="https://updates.fastdevs.co/"
```
Comment (brikr) [inline index.mdx:66]: is fastdevs.co still sticking around?

File: developer-portal/fast-api-fastjs.md [hunk 2]
```diff
@@ -1,102 +1,125 @@
...
+| **variant_id**      | Optional | variant_id represents a unique sub-idenfifier for this product. These are sometimes used to denote size / color, etc.                                                                                                            |
+| **quantity**        | Optional | quantity is the number of items that should be purchased.                                                                                                                                                                        |
+| **currency**        | Optional | currency is the string value of currency for the purchased.                                                                                                                                                                      |
+| **disabled**        | Optional | disbaled will prevent the button from fucntioning and show a disbaled state.                                                                                                                                                     |
```
Comment (brikr) [inline developer-portal/fast-api-fastjs.md:29]:
```suggestion
| **disabled**        | Optional | disabled will prevent the button from functioning and show a disabled state.                                                                                                                                                     |
```

File: developer-portal/fast-api-fastjs.md [hunk 3]
```diff
@@ -1,102 +1,125 @@
...
+| **product_options** | Optional | product_options are a set of configurations that further describe the product being ordered. It is a key value map. It must be a valid JSON object. The key is the selection options and value is the choice for that selection. |
+| **variant_id**      | Optional | variant_id represents a unique sub-idenfifier for this product. These are sometimes used to denote size / color, etc.                                                                                                            |
+| **quantity**        | Optional | quantity is the number of items that should be purchased.                                                                                                                                                                        |
+| **currency**        | Optional | currency is the string value of currency for the purchased.                                                                                                                                                                      |
```
Comment (brikr) [inline developer-portal/fast-api-fastjs.md:28]:
```suggestion
| **currency**        | Optional | currency is the string value of currency for the purchase.                                                                                                                                                                      |
```

File: developer-portal/fast-api-fastjs.md [hunk 4]
```diff
@@ -1,102 +1,125 @@
...
+
 ```jsx
 <script src="https://js.sandbox.fast.co/fast.js"></script>
 ```
 
-## Use the Fast Checkout button
+## Use the Fast Checkout button - Standard Approach
 
-### Render the button on your page
+Properties can be added to allow for integration without Javascript.
```
Comment (brikr) [inline developer-portal/fast-api-fastjs.md:19]:
```suggestion
Properties can be added to allow for integration without JavaScript.
```

**Review by brikr — CHANGES_REQUESTED** [review]
Thanks for doing this! Two other things apart from the comments I left in-line:
1. The structure of the fast.js docs is a little bit off. I think you can ditch the `Open Fast Checkout when the button is clicked` sub-header, and the `Handle events from Fast Checkout` header should be top-level as it works with both methods of initiating checkout.
2. We decided a bit after these docs were written that we would refer to the library as "fast.js" in all cases code or otherwise. Can you do a pass through the docs and remove any "Fast.js" or `fast.js` (inline code block), and replace with just "fast.js"?

File: index.mdx (hunk 1 above)
Comment (fasterSean) [reply index.mdx:66]: I'm not sure let me check

File: developer-portal/fast-api-fastjs.md [hunk 5]
```diff
@@ -122,11 +120,78 @@ checkoutButton.addEventListener("click", event => {
 
 🚨 It is important that you call `fast.checkout` synchronously within the `click` event handler. Many browsers will block new windows like Fast Checkout unless they are clearly connected to a user action such as clicking a button. If you perform asynchronous logic that takes several seconds, or call checkout in code in a different frame or window, the browser will prevent Fast Checkout from opening.
 
-### Handle events from Fast Checkout
+## Handle events from Fast Checkout
+
+Once the user has clicked the Fast Checkout button, the Fast popup window will have launched and the user can continue with their checkout. While this is going on, you can listen for fast.js `postMessage` events if you want to keep up with the status of the order on your frontend.
+
+- `Buy Now - Button Clicked` is emited when "Fast Checkout" button clicked on poduct page.
+- `Checkout Cart - Button Clicked` is emited when "Fast Checkout" button clicked on cart page.
```
Comment (brikr) [inline developer-portal/fast-api-fastjs.md:128]:
```suggestion
- `Buy Now - Button Clicked` is emitted when the "Fast Checkout" button is clicked on the product page.
- `Checkout Cart - Button Clicked` is emitted when the "Fast Checkout" button is clicked on the cart page.
```

File: developer-portal/fast-api-fastjs.md [hunk 6]
```diff
@@ -122,11 +120,78 @@ checkoutButton.addEventListener("click", event => {
...
+        }
+      ]
+    }
+  ]
+}
+```
+
+- `Checkout - Order Created` is emited when the order has been submitted through Fast checkout.
+- `Checkout - Order Updated` is emited when the order has changed while inside the Fast checkout window
+- `Checkout - Order Cancelled` is emited when the order has been cancelled from inside the Fast checkout window.
```
Comment (brikr) [inline developer-portal/fast-api-fastjs.md:156]:
```suggestion
- `Checkout - Order Created` is emitted when the order has been submitted through Fast Checkout.
- `Checkout - Order Updated` is emitted when the order has changed while inside the Fast Checkout window.
- `Checkout - Order Cancelled` is emitted when the order has been canceled from inside the Fast Checkout window.
```

File: developer-portal/fast-api-fastjs.md [hunk 7]
```diff
@@ -122,11 +120,78 @@ checkoutButton.addEventListener("click", event => {
...
+
+These event emit:
```
Comment (brikr) [inline developer-portal/fast-api-fastjs.md:158]:
```suggestion
These events emit:
```

File: developer-portal/fast-api-fastjs.md [hunk 8]
```diff
@@ -122,11 +120,78 @@ checkoutButton.addEventListener("click", event => {
...
+    }
+  }
+}
+```
 
-Once the user has clicked the Fast Checkout button, the Fast popup window will have launched and the user can continue with their checkout. While this is going on, you can listen for Fast.js `postMessage` events (link coming soon for Events list) if you want to keep up with the status of the order on your frontend.
+**Note**: All event properties have the potential to be undefined. You will need to check for undefined properties and values
 
-This is useful if you wish to forward data to your analytics provider, or if you want to do things like redirect to an order complete page after the user has finished their order and closed the Fast window.
+These events are useful if you wish to forward data to your analytics provider, or if you want to do things like redirect to an order complete page after the user has finished their order and closed the Fast window.
```
Comment (brikr) [inline developer-portal/fast-api-fastjs.md:194]: can you update the sample event watcher below to use an event that is real? 😅

File: developer-portal/fast-api-reference-fast-js.md [hunk 9]
```diff
@@ -1,24 +1,22 @@
...
+Throughout the login/checkout flow, fast.js dispatches events to the host page that can be used to watch for and perform certain actions with, such as sending relevant data to Google Analytics.
 
 # Event object
 
 Today, all events have a `name` field that denotes the type of event that fired (e.g. “Checkout - Order Created”), and some events have a `properties` field that contains special data relevant to the event that fired. For example, “Checkout - Order Created” events have the `order_id`, `order_items`, and `total` fields inside of their `properties` map.
 
-# Event list 
+# Event list
 
 These are all of the events that are currently supported, when they are sent, and what extra properties they contain.
```
Comment (brikr) [inline developer-portal/fast-api-reference-fast-js.md:11]: should this link to that event list on the other page?

**Review by brikr — CHANGES_REQUESTED** [review]
mostly grammar

**Review by brikr — APPROVED** [review]

**Pull request merged**